### PR TITLE
doc change on amp-accordion

### PR DESCRIPTION
**Pull request by zhouyx**
doc update for #3813 
DO NOT MERGE until `disable-session-states` attribute is live.
Need to wait for #6391 #3818 in PROD
cc @camelburrito @honeybadgerdontcare

### CODE REVIEW

File: extensions/amp-accordion/amp-accordion.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ Each of the `amp-accordion` component’s immediate children is considered a sec
 - The first child (of the section) must be one of `h1`, `h2`, ..., `h6`, `header`, and is the heading of the section.
 - The second child (of the section) can be any tag allowed in AMP HTML and is the content of the section.
 - Clicking/tapping on the heading of a section expands/ or collapses the section.
+- The collapsed/expanded state of each section in `amp-accordion` element will be preserved at session level. User has the option to disable this feature.
```
Comment (bpaduch) [inline extensions/amp-accordion/amp-accordion.md]: - After "in", add "the" 
- Before "session", add "the".
- Before "User", add "The" and change "User" to "user".

File: extensions/amp-accordion/amp-accordion.md [hunk 2]
```diff
@@ -70,6 +71,10 @@ Example:
 
 ## Attributes
 
+**disable-session-states**
+
+The `disable-session-states` attribute can be set on `<amp-accordion>` to opt out of preserving state of `amp-accordion` element.
```
Comment (bpaduch) [inline extensions/amp-accordion/amp-accordion.md]: Before "state", add "the".  Before "of" add "the".

Comment (ericlindley-g) [inline extensions/amp-accordion/amp-accordion.md]: w/ @bpaduch 's edits this becomes: "The `disable-session-states` attribute can be set on `<amp-accordion>` to opt out of preserving the state of the `amp-accordion` element."

I would add the clarification that this refers to the expanded/collapsed state. I suggest:

"The `disable-session-states` attribute can be set on `<amp-accordion>` to opt out of preserving the collapsed/expanded state of the `amp-accordion` element."

**Comment by zhouyx** [comment]
Thanks for the feedback. I have updated accordingly.

File: extensions/amp-accordion/amp-accordion.md [hunk 3]
```diff
@@ -48,6 +48,7 @@ Each of the `amp-accordion` component’s immediate children is considered a sec
 - The first child (of the section) must be one of `h1`, `h2`, ..., `h6`, `header`, and is the heading of the section.
 - The second child (of the section) can be any tag allowed in AMP HTML and is the content of the section.
 - Clicking/tapping on the heading of a section expands/ or collapses the section.
+- The collapsed/expanded state of each section in the `amp-accordion` element will be preserved at the session level. The user has the option to disable this feature.
```
Comment (camelburrito) [inline extensions/amp-accordion/amp-accordion.md]: The collapsed/expanded state of each section in the `amp-accordion` element will be preserved for the session. The user has the option to opt out of this by adding the `disable-session-states` attribute to the `amp-accordion` tag.

CC @bpaduch

**Comment by camelburrito** [comment]
@zhouyx - can we merge this now?

**Comment by zhouyx** [comment]
are we setting `amp-accordion-session-state-optout` tag to 1?

**Comment by zhouyx** [comment]
: ) OK. Anyone know adding to prod/canary-config.json on git is enough? Is the config sync now?

File: extensions/amp-accordion/amp-accordion.md (hunk 3 above)
Comment (zhouyx) [inline extensions/amp-accordion/amp-accordion.md]: changed.

**Comment by zhouyx** [comment]
@bpaduch Can you take another look and approve this? Thanks!

File: extensions/amp-accordion/amp-accordion.md [hunk 4]
```diff
@@ -48,6 +48,7 @@ Each of the `amp-accordion` component’s immediate children is considered a sec
 - The first child (of the section) must be one of `h1`, `h2`, ..., `h6`, `header`, and is the heading of the section.
 - The second child (of the section) can be any tag allowed in AMP HTML and is the content of the section.
 - Clicking/tapping on the heading of a section expands/ or collapses the section.
+- The collapsed/expanded state of each section in the `amp-accordion` element will be preserved for the session level. The user has the option to opt out of this by adding `disable-session-states` attribute to the `amp-accordion` tag.
```
Comment (bpaduch) [inline extensions/amp-accordion/amp-accordion.md]: Add "the"  before `disable-session-states`

**Comment by erwinmombay** [comment]
@zhouyx as long as it is in master it will be picked up

**Comment by zhouyx** [comment]
Cool! The config change was merged yesterday. And when can I expect the percentage to 100% in PROD? @erwinmombay

**Comment by zhouyx** [comment]
change in PROD, merged

**Pull request merged**